shared/cfg/guest-hw.cfg

```diff
@@ -82,23 +75,6 @@ variants:
         # Add -drive ...boot=yes unless qemu-kvm is 0.12.1.2 or newer
         # then kvm_vm will ignore this option.
         image_boot=yes
-        pseries:
```

Comment:
Only or No "pseries" are used for some powerpc special test,  please ensure this kind of test works in tp-qemu before merge this  PR.